core/client/verify_test.go

```diff
@@ -19,39 +19,15 @@ import (
 	"crypto"
 	"testing"
 
-	"github.com/google/keytransparency/core/crypto/vrf/p256"
-	"github.com/google/keytransparency/core/fake"
-
 	"github.com/google/trillian"
-	"github.com/google/trillian/client"
-	"github.com/google/trillian/crypto/keys/pem"
-	"github.com/google/trillian/merkle/hashers"
 
 	"github.com/golang/protobuf/ptypes"
 	"github.com/golang/protobuf/ptypes/any"
 
 	pb "github.com/google/keytransparency/core/api/v1/keytransparency_proto"
 	tcrypto "github.com/google/trillian/crypto"
-	_ "github.com/google/trillian/merkle/coniks" // Register coniks
-)
-
-var (
-	domainID = "default"
-	VRFPub   = []byte(`-----BEGIN PUBLIC KEY-----
-MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAE5AV2WCmStBt4N2Dx+7BrycJFbxhW
-f5JqSoyp0uiL8LeNYyj5vgklK8pLcyDbRqch9Az8jXVAmcBAkvaSrLW8wQ==
------END PUBLIC KEY-----`)
-	// openssl ecparam -name prime256v1 -genkey -out p256-key.pem
-	testPrivKey1 = `-----BEGIN EC PRIVATE KEY-----
-MHcCAQEEIBoLpoKGPbrFbEzF/ZktBSuGP+Llmx2wVKSkbdAdQ+3JoAoGCCqGSM49
-AwEHoUQDQgAE+xVOdphkfpEtl7OF8oCyvWw31dV4hnGbXDPbdFlL1nmayhnqyEfR
-dXNlpBT2U9hXcSxliKI1rHrAJFDx3ncttA==
------END EC PRIVATE KEY-----`
-	// openssl ec -in p256-key.pem -pubout -out p256-pubkey.pem
-	testPubKey1 = `-----BEGIN PUBLIC KEY-----
-MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAE+xVOdphkfpEtl7OF8oCyvWw31dV4
-hnGbXDPbdFlL1nmayhnqyEfRdXNlpBT2U9hXcSxliKI1rHrAJFDx3ncttA==
------END PUBLIC KEY-----`
+	_ "github.com/google/trillian/merkle/coniks"    // Register hasher
```

Comment:
Have an empty line separator before the anonymous imports.

Reply:
done